tools/model_tools/src/openvino/model_zoo/_configuration.py

```diff
@@ -95,6 +98,13 @@ def deserialize(cls, model, name, subdirectory, composite_model_name):
             framework = validation.validate_string_enum('"framework"', model['framework'],
                 _common.KNOWN_FRAMEWORKS.keys())
 
+            input_info = []
+            for input in model.get('input_info', []):
+                input_name = validation.validate_string('"input name"', input['name'])
+                shape = validation.validate_nonnegative_int_list('"input shape"', input.get('shape', []))
```

Comment:
why do you need this check?
-1 used for specifying dynamic shape for MO conversion, so it is valid value if we speak about shape in general understanding, if we want to have only static shapes, then yes, 1 is minimal possible value

Reply:
Removed the check